utils/shared-mime-info/Makefile

```diff
@@ -50,7 +59,8 @@ endef
 define Package/shared-mime-info/postinst
 #!/bin/sh
 
-update-mime-database /usr/share/mime/
+update-mime-database $${IPKG_INSTROOT}/usr/share/mime/
 endef
 
 $(eval $(call BuildPackage,shared-mime-info))
+$(eval $(call HostBuild))
```

Comment:
I would separate adding HostPackage into the separate commit, because commit says that it fixes post install behavior.